sentences/ca/light_HassTurnOn.yaml

```diff
@@ -6,6 +6,7 @@ intents:
         - "<engega> tots els llums [<area>]"
         - "<engega> totes les llums [<area>]"
         - "<engega> [els | les] llums [<area>]"
+        - "<engega> [el | la] llum [<area>]"
```

Comment:
"la llum" `the light` no pot ser "totes" (`all`), hauries de separar-ho en una altre secció.

Reply:
Entès, gràcies.